classes/scripts/runner.php

```diff
@@ -16,6 +16,8 @@ class runner extends atoum\script\configurable
 {
 	const defaultConfigFile = '.atoum.php';
 	const defaultBootstrapFile = '.bootstrap.atoum.php';
+	const defaultAutoloaderFile = '.autoloader.atoum.php';
+	const defaultComposerAutoloaderFile = 'vendor/autoload.php';
```

Comment:
what happens if we run atoum from a directory that does not contains vendor ? (for example, il we launch if from the tests dir with `../vendor/bin/atoum` ? does the includer will find the autoload ?